defaults/main.yml

```diff
@@ -41,16 +41,67 @@ docker__distribution_release: '{{ ansible_local.core.distribution_release
 docker__upstream: False
 
                                                                    # ]]]
+# .. envvar:: docker__variant [[[
+#
+# For upstream repositories the variant to be installed: ce or ee. Note that Docker EE
+# is not supported on Debian.
+docker__variant: 'ce'
+
+                                                                   # ]]]
+# .. envvar:: docker__channel [[[
+#
+# For upstream repositories choose the stable or edge channel.
+docker__channel: 'stable'
```

Comment:
Thanks for your remarks! I'll look into the trusty issues. 
Edition is probably the better naming instead of variant. Docker uses "variant" in their documentation, but also "edition": https://docs.docker.com/engine/installation/linux/debian/#prerequisites.
I'll leave channel as Docker uses this in their documentation to indicate the repository: https://docs.docker.com/engine/installation/.